src/components/app/ListOfPublishedProfiles.js

```diff
@@ -128,9 +146,25 @@ class PublishedProfile extends React.PureComponent<
     this.setState({ confirmDialogIsOpen: true });
   };
 
+  _renderPossibleErrorMessage() {
+    const { error } = this.state;
+    if (!error) {
+      return null;
+    }
+
+    return (
+      <p className="publishedProfilesDeleteError">
+        An error happened while deleting this profile.{' '}
+        <a href="javascript:void" title={error.message}>
+          Hover to know more.
+        </a>
+      </p>
```

Comment:
Because this is hard for you to check how this looks, here is a screenshot:
![image](https://user-images.githubusercontent.com/454175/93912782-79992e80-fd04-11ea-927f-6cf5de1d5fe3.png)